chapter_preliminaries/probability.md

```diff
@@ -1,304 +1,750 @@
+the (unknown) value of a different attribute. 
+Whenever we sample an individual from the population,
+we observe a realization of each of the random variables.
+Because the values taken by random variables 
+correspond to subsets of the sample space 
+that could be overlapping, partially overlapping, 
+or entirely disjoint,
+knowing the value taken by one random variable
+can cause us to update our beliefs 
+about what values of another random variable are likely.
+If a patient walks into a hospital 
+and we observe that they 
+are having trouble breathing
+and have lost their sense of smell,
+then we believe that they are more likely
+to have COVID than we might 
+if they had no trouble breathing
+and a perfectly ordinary sense of smell.
+
+
+When working with multiple random variables,
+we can construct events corresponding 
+to every combination of values 
+that the variables can jointly take.
+The probability function that assigns
+probabilities to each of these combinations
+(e.g. $A=a$ AND $B=b$),
+is called the *joint probability* function
+and simply returns the probability assigned 
+to the intersection of the corresponding subsets
+of the sample space. 
+The *joint probability* assigned to the event 
+where random variables $A$ and $B$ 
+take values $a$ and $b$, respectively,
+is denoted $P(A = a, B = b)$,
+where the comma indicates "AND". 
+Note that for any values $a$ and $b$,
+$P(A=a, B=b) \leq P(A=a)$ 
+and $P(A=a, B=b) \leq P(B = b)$,
+since for $A=a$ and $B=b$ to happen,
+$A=a$ has to happen *and* $B=b$ also has to happen.
+Interestingly, the joint probability
+tells us all that we can know about these
+random variables in a probabilistic sense,
+and can be used to derive many other
+useful quantities, including recovering the 
+individual distributions $P(A)$ and $P(B)$.
+To recover $P(A=a) $we simply sum up 
+$P(A=a, B=v)$ over all values $v$ 
+that the random variable $B$ can take:
+$P(A=a) = \sum_v P(A=a, B=v)$.
+
+
+The ratio $\frac{P(A=a, B=b)}{P(A=a)} \leq 1$
+turns out to be extremely important.
+It is called the *conditional probability*,
+and is denoted via the "|" symbol,
+$P(A=a|B=b) = P(A=a,B=b)/P(A=a)$
+and it tells us the new probability
+associated with the event $B=b$,
+once we condition on the fact $A=a$ took place.
+We can think of this conditional probability
+as restricting attention only to the subset
+of the sample space associated with $A=a$
+and then renormalizing so that
+all probabilities sum to 1.
+Conditional probabilities 
+are in fact probabilities
+and thus respect all of the axioms,
+so long as we condition all terms 
+on the same event and thus 
+restrict attention to the same sample space. 
+For instance, for disjoint events 
+$\mathcal{B}$ and $\mathcal{B}'$, we have that 
+$P(\mathcal{B} \cup \mathcal{B}'|A = a) = P(\mathcal{B}|A = a) + P(\mathcal{B}'|A = a)$. 
+
+
+Using the definition of conditional probabilities, 
+we can derive the famous result called *Bayes' Rule*.
```

Comment:
Bayes' Rule -> Bayes' theorem
(for term consistency in later sections)

Reply:
sure